front/src/applications/operationalStudies/views/SimulationResults.tsx

```diff
@@ -158,33 +155,36 @@ const SimulationResults = ({
       </div>
 
       {/* TRAIN : SPACE SPEED CHART */}
-      {selectedTrainRollingStock && trainSimulation && pathProperties && selectedTrainSchedule && (
-        <div className="osrd-simulation-container speedspacechart-container">
-          <div
-            className="chart-container"
-            style={{
-              height: `${speedSpaceChartContainerHeight + HANDLE_TAB_RESIZE_HEIGHT}px`,
-            }}
-          >
-            <SpeedSpaceChartContainer
-              trainSimulation={trainSimulation}
-              selectedTrainPowerRestrictions={selectedTrainPowerRestrictions}
-              rollingStock={selectedTrainRollingStock}
-              pathProperties={pathProperties}
-              heightOfSpeedSpaceChartContainer={speedSpaceChartContainerHeight}
-              setHeightOfSpeedSpaceChartContainer={setSpeedSpaceChartContainerHeight}
-            />
+      {selectedTrainRollingStock &&
+        trainSimulation?.status === 'success' &&
```

Comment:
This should be enough because `useSimulationResults` returns only the success simulation (or undefined). Same for the map and output table.

```suggestion
        trainSimulation &&
```

Reply:
Done

Reply:
Thank you. Could you add a description in this new commit to explain why we don't make these checks anymore ?